student.rb

```diff
@@ -1,4 +1,4 @@
-require_relative('./person')
+require_relative 'person'
 
 class Student < Person
   def initialize(classroom, age, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] Great job so far. However, there's little bug here. While trying to create student I got below error message. Kindly look into it and fix.
- [ ] You might need to check for others too.

### Error
![error](https://github.com/fredo509/OOP-school-library/assets/26597432/0f0bf001-9106-417f-b027-e21c582272f1)